plugins/modules/cloud/scaleway/scaleway_database_backup.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/python
+#
+# Scaleway database backups management module
+#
+# Copyright (C) 2020 Guillaume Rodriguez (g.rodriguez@opendecide.com).
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_database_backup
+short_description: Scaleway database backups management module
+author: Guillaume Rodriguez (@guillaume_ro_fr)
+description:
+    - This module manages database backups on Scaleway account
+      U(https://developer.scaleway.com)
```

Comment:
```suggestion
    - This module manages database backups on Scaleway account U(https://developer.scaleway.com).
```
I'd move the reference to `seealso:` section. Anyway the dot should be there (we shouldn't put it in `short_description` but should in `description`)